includes/definitions/discovery/sophos-xg.yaml

```
@@ -0,0 +1,12 @@
+mib: SOPHOS-XG-MIB
+modules:
+   os:
+      version:
+         - .1.3.6.1.4.1.2604.5.1.1.3.0
```

you should use the name instead, like so;
```suggestion
      version: SOPHOS-XG-MIB::sfosDeviceFWVersion
```